eagle-hadoop-metric/src/main/webapp/app/apps/hadoop_metric/ctrls/regionDetailCtrl.js

```diff
@@ -0,0 +1,179 @@
+						},
+						areaStyle: {normal: {}},
+						yAxis: [{
+							axisLabel: {
+								formatter: function (value) {
+									return common.number.sizeFormat(value, 0);
+								}
+							}
+						}]
+					};
+
+					var gctimeoption = {
+						legend: {
+							x: 'center', y: 'bottom'
+						},
+						yAxis: [{
+							axisLabel: {
+								formatter: function (value) {
+									return value / 1000 + ' S';
+								}
+							}
+						}]
+					};
+					$scope.metricList = [];
+					$scope.metricList.push(mergeSeries("Memory Usage", [res[0], res[1]], ["nonheap", "heap"], sizeoption));
+					$scope.metricList.push(mergeSeries("Direct Memory Usage", [res[2]], ["directmemory"], sizeoption));
+					$scope.metricList.push(mergeSeries("GC count", [res[3]], ["GC count"], {}));
+					$scope.metricList.push(mergeSeries("GC TimeMillis", [res[4]], ["GC TimeMillis"], gctimeoption));
+					$scope.metricList.push(mergeSeries("QueueSize", [res[5]], ["QueueSize"], {}));
+					$scope.metricList.push(mergeSeries("NumCallsInGeneralQueue", [res[6]], ["NumCallsInGeneralQueue"], {}));
+					$scope.metricList.push(mergeSeries("NumActiveHandler", [res[7]], ["NumActiveHandler"], {}));
+					$scope.metricList.push(mergeSeries("IPC Queue Time (99th)", [res[8]], ["IPC Queue Time (99th)"], {}));
+					$scope.metricList.push(mergeSeries("IPC Process Time (99th)", [res[9]], ["IPC Process Time (99th)"], {}));
+					$scope.metricList.push(mergeSeries("QueueCallTime_num_ops", [res[10]], ["QueueCallTime_num_ops"], {}));
+					$scope.metricList.push(mergeSeries("ProcessCallTime_num_ops", [res[11]], ["ProcessCallTime_num_ops"], {}));
+					$scope.metricList.push(mergeSeries("RegionCount", [res[12]], ["RegionCount"], {}));
+					$scope.metricList.push(mergeSeries("StoreCount", [res[13]], ["StoreCount"], {}));
+					$scope.metricList.push(mergeSeries("MemStoreSize", [res[14]], ["MemStoreSize"], sizeoption));
+					$scope.metricList.push(mergeSeries("StoreFileSize", [res[15]], ["StoreFileSize"], sizeoption));
+					$scope.metricList.push(mergeSeries("TotalRequestCount", [res[16]], ["TotalRequestCount"], {}));
+					$scope.metricList.push(mergeSeries("ReadRequestCount", [res[17]], ["ReadRequestCount"], {}));
+					$scope.metricList.push(mergeSeries("WriteRequestCount", [res[18]], ["WriteRequestCount"], {}));
+					$scope.metricList.push(mergeSeries("SlitQueueLength", [res[19]], ["SlitQueueLength"], {}));
+					$scope.metricList.push(mergeSeries("CompactionQueueLength", [res[20]], ["CompactionQueueLength"], {}));
+					$scope.metricList.push(mergeSeries("FlushQueueLength", [res[21]], ["FlushQueueLength"], {}));
+					$scope.metricList.push(mergeSeries("BlockCacheSize", [res[22]], ["BlockCacheSize"], sizeoption));
+					$scope.metricList.push(mergeSeries("BlockCacheHitCount", [res[23]], ["BlockCacheHitCount"], {}));
+					$scope.metricList.push(mergeSeries("BlockCacheCountHitPercent", [res[24]], ["BlockCacheCountHitPercent"], {}));
+
+					$scope.regionstatus = res[25];
```

Comment:
`regionstatus` not used as array in web template. May be just `$scope.regionstatus = (res[25] || [])[0]` is better?